packages/@pollyjs/adapter-http/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@pollyjs/adapter-http",
```

Comment:
We should rename this to `adapter-node-http` just to denote that this a node only adapter.